### ISSUE TITLE
Publicly show date material was hidden from public view

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#5471]
If a message is hidden from public view readers should see when it was hidden. 

The date of any report might be relevant context too. 

details/request/

does already show 

report_request and edit_incoming events but "edit_incoming" isn't described, where appropriate, in plain English as "hidden" and that detail is perhaps too deep and could be presented on the correspondence thread when material has been hidden.

**Comment by garethrees** [comment]
This would be nice as part of #4565

**Comment by garethrees** [comment]
Attributes changed are now shown in event history as of https://github.com/mysociety/alaveteli/pull/7892
